content/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists.md

```diff
@@ -29,7 +29,7 @@ Secret gists don't show up in {% data variables.gists.discover_url %} and are no
 
 If your site administrator has disabled private mode, you can also use anonymous gists, which can be public or secret.
 
-{% data reusables.gist.anonymous-gists-cannot-be-deleted %}
+{% data reusables.gist.anonymous-gists-cannot-be-deleted %}.
```

Comment:
```suggestion
{% data reusables.gist.anonymous-gists-cannot-be-deleted %}
```

This doesn't need a period at the end as it's referencing content that already has one:

https://github.com/github/docs/blob/main/data/reusables/gist/anonymous-gists-cannot-be-deleted.md

Reply:
sounds good. I think Prettier or Markdownlint extensions have made those additions without me noticing - will be more vigilant on that 👍